lane_detection_node: 
  ros__parameters: 
    Hue_low : 47 
    Hue_high : 150 
    Saturation_low : 181 
    Saturation_high : 222 
    Value_low : 71 
    Value_high : 118 
    number_of_lines : 100 
    error_threshold : 0.21 
    Width_min : 5 
    Width_max : 500 
    gray_lower : 0 
    inverted_filter : 0 
    camera_start_height : 300 
    camera_bottom_height : 600 
    camera_left_width : 0 
    camera_right_width : 800 
calibration_node: 
  ros__parameters: 
    Hue_low : 47 
    Hue_high : 150 
    Saturation_low : 181 
    Saturation_high : 222 
    Value_low : 71 
    Value_high : 118 
    number_of_lines : 100 
    error_threshold : 0.21 
    Width_min : 5 
    Width_max : 500 
    gray_lower : 0 
    inverted_filter : 0 
lane_guidance_node: 
  ros__parameters: 
    steering_sensitivity : 1.0 
    zero_throttle  : 0.4650000000000001 
    zero_error_throttle : -0.248 
    error_throttle : 0.0 
    error_threshold : 0.21 
vesc_twist_node: 
  ros__parameters: 
    max_rpm : 10000 
    steering_polarity : 1 
    throttle_polarity : 1 
adafruit_twist_node: 
  ros__parameters: 
    steering_polarity : 1 
    throttle_polarity : 1 
